generate_api.py

```diff
@@ -0,0 +1,12 @@
+import requests
```

Comment:
Not entirely necessary, but if we decide to generate using the API, this is the code to submit it.